scripts/qa_crowbarsetup.sh

```diff
@@ -3260,6 +3260,8 @@ function addfloatingip
     nova floating-ip-create | tee floating-ip-create.out
     floatingip=$(perl -ne "if(/\d+\.\d+\.\d+\.\d+/){print \$&}" floating-ip-create.out)
     nova add-floating-ip "$instanceid" "$floatingip"
+    # flush arp tables in case this floating ip was recently used for something else
+    arp -d "$floatingip"
```

Comment:
Isn't neutron supposed to issue a gratuitous ARP to make this kind of thing unnecessary?  Or is there a reason it's not a neutron bug?  I've seen it do that in other situations.